packages/core/plugins/core-plugins-server-internal/src/legacy_config.test.ts

```diff
@@ -5,37 +5,17 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-
 import { take } from 'rxjs/operators';
-import { ConfigService, Env } from '@kbn/config';
-import { getEnvOptions, rawConfigServiceMock } from '@kbn/config-mocks';
 import { getGlobalConfig, getGlobalConfig$ } from './legacy_config';
-import { REPO_ROOT } from '@kbn/utils';
-import { loggingSystemMock } from '@kbn/core-logging-server-mocks';
 import { duration } from 'moment';
 import { fromRoot } from '@kbn/utils';
 import { ByteSizeValue } from '@kbn/config-schema';
-import { Server } from '../server';
```

Comment:
`Server` isn't accessible from within the package and we can't use that to manage the config.